core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -952,18 +919,13 @@ object ConfigCommand extends Logging {
       val hasEntityDefault = entityNames.exists(_.isEmpty)
 
       val numConnectOptions = (if (options.has(bootstrapServerOpt)) 1 else 0) +
-        (if (options.has(bootstrapControllerOpt)) 1 else 0) +
-        (if (options.has(zkConnectOpt)) 1 else 0)
+        (if (options.has(bootstrapControllerOpt)) 1 else 0)
       if (numConnectOptions == 0)
-        throw new IllegalArgumentException("One of the required --bootstrap-server, --boostrap-controller, or --zookeeper arguments must be specified")
+        throw new IllegalArgumentException("One of the required --bootstrap-server or --bootstrap-controller arguments must be specified")
       else if (numConnectOptions > 1)
-        throw new IllegalArgumentException("Only one of --bootstrap-server, --boostrap-controller, and --zookeeper can be specified")
-
-      if (options.has(allOpt) && options.has(zkConnectOpt)) {
-        throw new IllegalArgumentException(s"--bootstrap-server must be specified for --all")
-      }
-      if (options.has(zkTlsConfigFile) && !options.has(zkConnectOpt)) {
-        throw new IllegalArgumentException("Only the --zookeeper option can be used with the --zk-tls-config-file option.")
+        throw new IllegalArgumentException("Only one of --bootstrap-server or --bootstrap-controller can be specified")
+      if (options.has(zkTlsConfigFile)) {
```

Comment:
The `zkTlsConfigFile` option should be removed as well.

Reply:
I've made changes based on your comments.